docs/resources/project.md

```diff
@@ -128,3 +132,6 @@ Required:
 Optional:
 
 - **description** (String)
+
+## Note
+- Project provider has limitations assigning large number of repositories through `repos` attribute.    
```

Comment:
This is not adequate for our users. Please provide more details for workaround(s). For example:
- what's the error the users will see?
- how to use the cli to limit parallel processing
- how many repos is the rough limit where this issue occurs

Put yourself in the shoes of our users who has no idea about this limitation. What information will help them move forward with using this provider?

Reply:
updated the text. @alexhung can you please review the text.